### ISSUE TITLE
the value for the available_actions

### ISSUE THREAD

**Issue by EddyJason** [marlbenchmark/on-policy#71]
why the value of the available_actions is the array of (12800, 5) and the value of each row is [1, 1, 1, 1, 1]. 1 simply indicate whether or not it simply indicates whether or not it is moving, but does not consider the direction of movement.

**Comment by zoeyuchao** [comment]
Okay. You have created so many issues and I can not figure out which one I should answer, could you summarize your questions clearly? Chinese is okay.

**Comment by EddyJason** [comment]
我想知道动作空间中0与1的区别，0是否代表不动，1是否代表移动

**Comment by EddyJason** [comment]
一个新的问题是：为什么动作空间是5位数组？其中四位应该是东南西北，但是第五位是序号吗？
